keplergl_cli/keplergl_cli.py

```diff
@@ -40,14 +40,21 @@ def __init__(
             names=None,
             read_only=False,
             api_key=None,
-            style=None):
+            style=None,
+            config_file=None,
+            output_map=None,
+            open_browser=False):
```

Comment:
I originally didn't intend `open_browser` to be exposed through `__init__`. The rationale being that passing `data` into here is just a shortcut for using `render` directly, and we don't need to expose all the same params